ts/components/services/ServicesSectionsList.tsx

```diff
@@ -95,6 +99,23 @@ class ServicesSectionsList extends React.PureComponent<Props> {
     );
   }
 
+  // View for empty list in National and All tabs
+  private othersListEmptyComponent() {
+    return (
+      !this.props.isLocal && (
```

Comment:
```suggestion
     
```
The check to render it only when it is not local it already done in **_renderList_** metod